std/prim/char.pol

```diff
@@ -0,0 +1,3 @@
+/// Unicode code points according to definitions D9 and D10 of the
```

Comment:
I think we should follow Rust in making `Char` contain UTF-32 scalar values (= set of codepoints without [surrogates](https://en.wikipedia.org/wiki/Universal_Character_Set_characters#Surrogates) which are irrelevant for the UTF-32 representation). See discussion on https://github.com/polarity-lang/polarity/pull/579.

Reply:
No, we want D76:

> D76 Unicode scalar value: Any Unicode code point except high-surrogate and low-surro-
gate code points.
> • As a result of this definition, the set of Unicode scalar values consists of the
ranges 0 to D7FF 16 and E000 16 to 10FFFF 16, inclusive.